app/legacy/v2/notifications/route_schema.py

```diff
@@ -185,7 +228,7 @@ class V2PostSmsRequestModel(V2PostNotificationRequestModel):
 
     @model_validator(mode='after')
     def phone_number_or_recipient_id(self) -> Self:
-        """One, and only one, of "phone_number" or "recipient_identifier" must not be None.
+        """At least one, of "phone_number" or "recipient_identifier" must not be None.
```

Comment:
somehow I am surprised that ruff didn't insist on single quotes inside this double-quoted string?